Sources/SwiftDocC/Model/Rendering/References/TopicRenderReference.swift

```diff
@@ -253,9 +241,9 @@ public struct TopicRenderReference: RenderReference, VariantContainer, Equatable
         case beta
         case deprecated
         case defaultImplementations
-        case titleStyle
-        case name
-        case ideTitle
+        case propertyListTitleStyle = "titleStyle"
```

Comment:
I understand this was done to preserve RenderJSON compatibility but I wonder if there is a path forward to eventually encoding these in a more sensible way?

Reply:
Definitely beyond the scope of this change, but I wonder if we should encode a RenderJSON version field or something.